.gitignore

```diff
@@ -31,3 +31,7 @@ bwapi/libReplayToolTest/TestData/Replay_Tool_Actual.exe
 bwapi/ipch/
 Release_Binary/Chaoslauncher/__log.txt
 Release_Binary/ExampleAIClient/Source/
+
+/bwapi/packages/*
+!/bwapi/packages/repositories.config

```

Comment:
Oh, the following `nuget/*.nupkg` line isn't supposed to be here, but the repositories.config and other nuget packages under `bwapi/packages/` are already included in bwapi:
https://github.com/bwapi/bwapi/blob/38343b0263e7de72f4a7ab66d897a879b4a97802/bwapi/packages/
If we add the nuget dependency on boost, we might as well also remove that directory altogether from the repo and depend on nuget for downloading the packages, right?